ext/standard/php_var.h

```diff
@@ -48,6 +48,8 @@ struct php_unserialize_data {
 	void *last;
 	void *first_dtor;
 	void *last_dtor;
+	void *first_wakeup;
```

Comment:
Could someone familiar with the php release process clarify something for me? I think that this might fall under incompatible ABI, and should instead be released in the 7.1 beta?

E.g.

1. An extension which uses php_serialize_data_t to call serialize is compiled against 7.0.9. A shared object is compiled.
2. If this changes in 7.0.10, then that extension would break in 7.1.10 if the shared object wasn't recompiled against 7.0.10 (The struct the extension allocates on the stack only has enough space for the first 4 parameters)

Or is it the case that shared objects are already expected to be recompiled every time the subversion changes?